validation-test/compiler_crashers_2_fixed/sr10062.swift

```diff
@@ -0,0 +1,6 @@
+// RUN: not %target-typecheck-verify-swift
```

Comment:
I think the crash was in silgen and not Sema. If the type checker starts allowing this again this test will still pass

Reply:
Hmm it hit an assertion in Sema for me, but only because I am running a build with asserts on. But yeah, otherwise it will crash in SILGen. I've now added `-emit-silgen` flag!